### PR TITLE
Init Yandex Cloud support (terraform-provider-yandex)

### PR DESCRIPTION
**Pull request by GennadySpb**
For resource types:
* Compute Instance
* Compute Disk
* VPC Subnet
* VPC Network

### CODE REVIEW

**Comment by GennadySpb** [comment]
@googlebot I signed it!

**Comment by sergeylanzman** [comment]
@GennadySpb Thanks!
Can you add licence notes to every new go file?

**Comment by GennadySpb** [comment]
Hi @sergeylanzman !
Are you ask about add https://pastebin.com/raw/0Ffz7Zth into each file?

**Comment by sergeylanzman** [comment]
yes, you can copy from other files

**Comment by GennadySpb** [comment]
@sergeylanzman  done  ;)

File: cmd/provider_cmd_yandex.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
+package cmd
```
Comment (sergeylanzman) [inline cmd/provider_cmd_yandex.go:1]: add license notes

Comment (GennadySpb) [reply cmd/provider_cmd_yandex.go:15]: Oops.. fixed

**Pull request merged**